css/mobile.css

```diff
@@ -0,0 +1,519 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Staatliches&display=swap');
+@import url(./css/style.css);
+
+:root {
+  --color-primary: #575757;
+  --color-dark: #272a31;
+  --color-accent: #ec5242;
+  --color-negative: #d3d3d3;
+  --color-white: #fff;
+  --ff-text: 'Lato', sans-serif;
+  --ff-title: 'COCOGOOSE', sans-serif;
+  --max-width: 75vw;
+}
+
+/* body * {
+  outline: 1px solid blue;
+} */
+
+/* Mobile Version */
+
+/* Land Page Section */
+
+@media screen and (max-width: 768px) {
+  #nav-bar-main {
+    display: none;
+  }
+
+  header #menu {
+    top: 3vw;
+    padding-left: 5vw;
+
+    /* color: var(--color-dark); */
+    align-items: center;
+    position: absolute;
+    background-color: transparent;
+  }
+
+  #land-page {
+    background-image: url(../images/mobile_intro_bg.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    height: 123vw;
+  }
+
+  /* Nav Bar Section */
+
+  #land-page-container {
```

Comment:
- [ ]  According to Microverse guidelines, please avoid using id as a selector. ID selectors have a higher specificity than attribute selectors but using ids in css isn't recommended and can't be reused. CSS classes can do everything IDs can.

Reply:
- [ ] As requested by the previous reviewer, make sure that you use class selectors instead of using ids selectors. you can do it just by replacing wherever you used id as a selector with class in your HTML code and then change the `#` with `.`  in your CSS.
Example:

HTML
```
 <div class="land-page-container">
      your code here....
    </div>
```
CSS
```
.land-page-container {
  your style here...
}
```
Kindly, do it wherever you did that mistake which is not a best practice.